src/components/TokenEnterAmount.tsx

```diff
@@ -48,6 +49,16 @@ export function formatNumber(value: string) {
     .replaceAll('_', groupingSeparator)
 }
 
+export function unformatNumberForProcessing(value: string) {
```

Comment:
@kathaypacific hmm, I'm still unsure on this one. It is only used in a single place so far to actually un-format already formatted number that comes from the input and format it for the processing purposes. I think it makes a more clear distinction for the specific purpose of the function:
- it takes specifically formatted number 
- it unformats and formats it specifically for processing

I've only made is a separate function to simplify description as otherwise I'd need to write a comment explaining what it does and why, which feels more concise.

In case of `formatNumberToStandard/parseNumberToStandardFormat ` - we need to define what we mean by "standard". And in case of `applyUSNumberFormatting` - we need to understand why we specifically use `US` number formatting (even though I'm not sure it's US specific as up to this point I thought this was a general format that everyone uses and was always using it myself 😄)

At least, that's my train of though. What do you think about it?